server/src/main/java/com/linecorp/centraldogma/server/internal/storage/project/DefaultProject.java

```diff
@@ -200,7 +207,7 @@ private void createReservedRepos(long creationTimeMillis, boolean useDogmaRepoAs
         }
         if (!useDogmaRepoAsMetaRepo && !repos.exists(REPO_META)) {
             try {
-                repos.create(REPO_META, creationTimeMillis, Author.SYSTEM);
+                repos.create(REPO_META, creationTimeMillis, Author.SYSTEM, false);
```

Comment:
Question) Why isn't `useDogmaRepoAsMetaRepo` used here?

```suggestion
                repos.create(REPO_META, creationTimeMillis, Author.SYSTEM, useDogmaRepoAsMetaRepo);
```